plugins/samson_ledger/test/initializers/ledger_test.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+require_relative "../test_helper"
+
+SingleCov.covered! file: 'plugins/samson_ledger/config/initializers/samson_ledger/client.rb'
```

Comment:
ah yes ... it assumes that tested files are in lib or app ... maybe move the code to lib instead ?